build/ci/github_actions/operator/deploy_driver.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -xel
+set +o pipefail
+
+operator_image_for_test=$operator_image_repository_for_test:$operator_image_tag_for_test
+
+kind_node_name=`docker ps --format "{{.Names}}"`
+docker exec -i $kind_node_name apt-get update
+docker exec -i $kind_node_name apt -y install open-iscsi
+
+edit_cr_images (){
```

Comment:
i've never [seen](https://github.com/icy/bash-coding-style/blob/v1.0.1/README.md) this parenthesis spacing convention before.
could you open a ticket to add [shellcheck](https://github.com/koalaman/shellcheck) linter to our scripts, and write its number here?

Reply:
CSI-3219